internal/engine/wazevo/backend/isa/amd64/instr.go

```diff
@@ -1880,6 +1892,21 @@ func (r roundingMode) String() string {
 	}
 }
 
+type cmpPredicate uint8
+
+const (
+	cmpPredicateEqualUnordered cmpPredicate = 8
+)
+
+func (r cmpPredicate) String() string {
+	switch r {
+	case cmpPredicateEqualUnordered:
+		return "equalUnordered"
+	default:
+		panic("unsupported")
+	}
+}
+
```

Comment:
instead of defining this at all, can you please use cond flag directly? in anyway this const is not used except for unit tests, which I don't think make sense as well.

Reply:
also please add comments that this is supposed to be used as the imm for XmmRmRImm.